src/pudl/package_data/phmsagas/column_maps/yearly_miles_of_pipe_by_class_location.csv

```diff
@@ -0,0 +1,66 @@
+year_index,1990,1991,1992,1993,1994,1995,1996,1997,1998,1999,2000,2001,2002,2003,2004,2005,2006,2007,2008,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2021,2022
+data_date,,,,,,,,,,,,,,,,,,,,,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of
+report_year,,,,,,,,,,,,yr,yr,yr,yr,yr,yr,yr,yr,yr,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year
+report_number,,,,,,,,,,,,rptid,rptid,rptid,rptid,rptid,rptid,rptid,rptid,rptid,report_number,report_number,report_number,report_number,report_number,report_number,report_number,report_number,report_number,report_number,report_number,report_number,report_number
+supplemental_report_number,,,,,,,,,,,,tr_sup_id,tr_sup_id,tr_sup_id,tr_sup_id,tr_sup_id,tr_sup_id,tr_sup_id,tr_sup_id,tr_sup_id,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number,supplemental_number
+operator_id_phmsa,,,,,,,,,,,,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id,operator_id
+operator_name_phmsa,,,,,,,,,,,,name,name,name,name,name,name,name,name,name,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp,parta2nameofcomp
+commodity_group,,,,,,,,,,,,,,,,,,,,,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity,parta5commodity
+interstate_or_intrastate,,,,,,,,,,,,,,,,,,,,,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra
+report_state,,,,,,,,,,,,stop,stop,stop,stop,stop,stop,stop,stop,stop,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name
+onshore_transmission_pipe_class_1_miles,,,,,,,,,,,,b4ton_1,b4ton_1,b4ton_1,b4ton_1,b4ton_1,b4ton_1,b4ton_1,b4ton_1,b4ton_1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1,partltonc1
```

Comment:
I think we are a little inconsistent about the ordering of onshore/offshore and transmission/gathering:

```
> ag ^onshore_ -c 
yearly_miles_of_pipe_by_class_location.csv:25
yearly_miles_of_transmission_pipe_by_nps.csv:35
yearly_miles_of_gathering_pipe_by_nps.csv:95
> ag ^transmission_onshore -c
yearly_transmission_gathering_summary_by_commodity.csv:1
> ag ^gathering_onshore -c 
yearly_transmission_gathering_summary_by_commodity.csv:4
```

It appears that "shoreyness" comes before "transmissioniness" in most cases, should we standardize that everywhere?